include/opentxs-proto/verify/VerifyStorage.hpp

```diff
@@ -91,21 +92,32 @@ static const VersionMap StorageNymAllowedHash = {
     {4, {2, 4}},
     {5, {2, 5}},
     {6, {2, 6}},
+    {7, {2, 6}},
 };
 static const VersionMap StorageNymAllowedBip44Account{
     {4, {1, 1}},
     {5, {1, 1}},
     {6, {1, 1}},
+    {7, {1, 1}},
 };
+static const VersionMap StorageNymAllowedBip47ChannelList = {
+    {7, {1, 1}}
+};
+
 static const VersionMap StorageNymAllowedBlockchainAccountList{
     {4, {1, 1}},
     {5, {1, 1}},
     {6, {1, 1}},
+    {7, {1, 1}},
 };
 static const VersionMap StorageNymListAllowedHash = {
```

Comment:
It shouldn't be necessary to change the version of StorageNymList

Reply:
Done